app-text/bibutils/bibutils-6.10.ebuild

```diff
@@ -30,6 +33,12 @@ src_compile() {
 		LDFLAGSIN="${LDFLAGS}"
 }
 
+src_install() {
+	export EINSTALLDIR="${ED}/usr/bin"
```

Comment:
Can’t we make it repeat DESTDIR instead (and libdir if necessary, but that should be passed in configure _ideally_)

Reply:
I am not totally sure what you meant, but I made the patches a bit less gross and just prepended DESTDIR to the install destinations.

Reply:
Oh, much better. Nice!